Worked on the instance rules and guidelines.

zackbatist opened the pull request:
Adapted the language to be more suited towards archaeology. Added encouragement to participate in content moderation. Added a stipulation to CW or tag announcement posts. Added a section on brand accounts, which includes some tentative ideas on how to proceed with them. Added a stub regarding uploaded media.

zackbatist committed:
Worked on the instance rules and guidelines.

Adapted the language to be more suited towards archaeology. Added encouragement to participate in content moderation. Added a stipulation to CW or tag announcement posts. Added a section on brand accounts, which includes some tentative ideas on how to proceed with them. Added a stub regarding uploaded media.

zackbatist committed:
Added a section on uploaded media

joeroe reviewed (changes requested):
Like it a lot so far! Some comments broken by section below. A lot is not stuff I necessarily disagree with or think we shouldn't do, but question whether it needs to be said explicitly. The longer things like this are the less people read them, and I think scholar.social errs on the side of saying too much.

## Rules

- [ ] I'm not convinced that bots or institutional accounts are damaging enough to disallow. Maybe we could ask them to be unlisted?
- [ ] Ditto for automated cross-posting... I actually like this kind of thing on Twitter. Maybe should also be opt in?
- [ ] "Unsolicited advice to strangers" – is this a big enough problem to be worth explicitly disallowing?
- [ ] Ditto for NFTs. On general instances, sure. But in archaeology?
- [ ] Ditto for shortened links. I don't think many people deliberately use these any more.

## Encouraged uses of archaeo.social

- [ ] I would prefer e.g. "share" or "summarise" publications over "brag"
- [ ] I feel like "recruiting people for surveys" is too niche to specifically mention?
- [ ] Off-topic stuff being unlisted could work, I guess. But I'd explain what "unlisted" means (i.e. followers can still see it).

## Who is welcome...

- [ ] The initial description of the instance was that it is for those "primarily interested in scholarly communication about archaeology". I'd like to stick to this, and understand it to include those who do not "identify" as an archaeologist but are simply interested in the subject.
- [ ] Do we really expect police officers to even try and sign up? And there are spooks in archaeology, for sure, but they're generally not too open about it...

## Content Warnings

- [ ] I think using CWs for 'announcements' should be optional, and would prefer to see what norms emerge organically in that regard rather than prescribe it now.

## Image descriptions

- [ ] Might be an idea to explain (or link to a good explanation of) what makes good alt text and why it's important.

## The local timeline

I like this a lot. We should encourage people to be conscious about curating the local timeline as a shared noticeboard for archaeology.

- [ ] Maybe some of the requests to mark things as unlisted above could be moved here for clarity
- [ ] I don't want to recommend or not recommend any apps. Statements like this sound like they're going to get out of date fast.
- [ ] "Not _every_ post on the Local Timeline should be CW'd, but most should" – really?

## Unwelcome conduct...

- [ ] See above for some of these
- [ ] Hate to be the "actually" guy, but a DOI is a stable identifier not a shortlink, and should be encouraged!

## Regarding attempts ...

- [ ] I'd also file this under things I hope will never come up

## Brand accounts

Okay yes, sounds good. Should change the text above to be in line with this.

## Uploaded media

- [ ] We should expand this into a broader statement about the instance's sustainability

## Moderated servers

I think this section is auto-generated by Mastodon?

zackbatist committed:
Updated the rules and guidelines.

Based on comments from https://github.com/archaeo-social/mastodon/pull/5#pullrequestreview-1188209204.

zackbatist commented:
I updated the draft based on your comments. I also created a section called "server administration and support" with a few bullet points, but you might be better suited for writing that part.
